### PR TITLE
chore: update `version-changed` to v2.0.0

### LINKED ISSUE #21: Please update `version-changed` to v2.0.0
v2.0.0 of [version-changed](https://github.com/ralphtheninja/version-changed) has a much-needed fix for us, using a relative path for a `package.json` (we happen to be running prebuilds from a deeply nested structure in a git repo). Could you please update `prebuild-ci`?

### PR DESCRIPTION
**Pull request by mbroadst**
fixes #21

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,14 +13,14 @@
...
-    "version-changed": "^1.1.1"
+    "version-changed": "^2.0.0"
   },
   "devDependencies": {
     "standard": "^13.0.1",
     "tape": "^4.10.1"
   },
   "engines": {
-    "node": ">=6"
+    "node": ">6"
```
Comment (vweevers) [inline package.json]:
```suggestion
    "node": ">=8"
```

Explicit is better.

**Comment by mbroadst** [comment]
@vweevers I actually just did a review of your dependencies and `version-changed` is the only one that specifies `engines` at all. Might I propose a more dramatic change here, where the functionality of `version-changed` is merged in, and `engines` is removed alltogether? We (at MongoDB) need to continue to support older versions of node, so this is actually kind of a detriment to us.

**Comment by vweevers** [comment]
For how long do you wish `prebuild-ci` to continue to support 6?

**Comment by vweevers** [comment]
Also note that `prebuild-ci` is not used by end users.

**Comment by mbroadst** [comment]
@vweevers my position is that you don't technically need to support node 6, but you don't need to _prevent_ users of node 6 from using your package. Most of your other dependencies take this position (e.g. `run-series`). I've created https://github.com/ralphtheninja/version-changed/pull/5 to address this with `version-changed`. 

I understand it's not used by end users, but we do run it in our CI which includes node 4 and 6 matrix variants.

**Comment by vweevers** [comment]
Not declaring the minimum version is out of the question for me. I'm fine with keeping node 6, but it should be explicitly declared (and tested in CI) because I don't want any vagueness. We can't half-support a version, it's either/or.

**Comment by mbroadst** [comment]
Well, please go ahead and merge this in. For the time being I guess I will need to fork your project, and hopefully rejoin in a few months when we fully drop support for non-LTS node versions.

**Comment by vweevers** [comment]
To be clear: we don't have to drop 6 here, if we restore support of 6 in `version-changed`.

**Comment by ralphtheninja** [comment]
LGTM

**Comment by lovell** [comment]
I'm happy to drop support for Node.js 6, which reached EOL in April last year. This would also allow for a semver-major update of other dependencies to their latest version e.g. `cross-spawn` and `standard`.

**Comment by vweevers** [comment]
The effort on our part to keep supporting node 6 is minimal. There is no good reason to drop it yet, but we do have at least one user wishing we keep it.

> This would also allow for a semver-major update of other dependencies to their latest version e.g. cross-spawn and standard.

These don't matter: `standard` is a devDependency and a noop on node 6, and `cross-spawn@6` works just fine for the time being.

**Pull request closed without merging**